src/main/java/ca/on/oicr/pineryreports/Main.java

```diff
@@ -67,9 +68,9 @@ public static void main(String[] args) {
         CommandLine reportCommand = getCommandLine(args, opts, true);
         report.processOptions(reportCommand);
         
-        LOG.info("Options ok. Generating " + report.getTitle() + "...");
+        LOG.info(String.format("Options ok. Generating %s...", report.getTitle()));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Invoke method(s) only conditionally. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)

Reply:
SLF4J's optimized version of `String#format` would make Sonar happy:

```
LOG.info("Options ok. Generating {}...", report.getTitle());
```

(not that it matters since this app is not really performance-sensitive)

Reply:
Ah, that's why it continued to complain. Thanks for the heads up!